lib/actions/ui.js

```diff
@@ -240,6 +240,7 @@ export const MobileScreens = {
  * (currently only used in batch results).
  */
 export const ItineraryView = {
+  /** FIXME: Duplicate value for 'list', DEFAULT and LIST */
   DEFAULT: 'list',
```

Comment:
@evansiroky and @binh-dam-ibigroup, do you see any reason to maintain `LIST` and `DEFAULT` (which both have `list` as their value)? I'm of the mind that we should maybe just entirely replace `DEFAULT` with `LIST`, but wanted to get your feedback in case there was a good reason to keep `DEFAULT`.

Reply:
The alternative would be to replace the DEFAULT entry with
```js
const DEFAULT_ITINERARY_VIEW = ItineraryView.LIST
```
(Whether the view state is default determines whether the `ui_itineraryView` parameter is added to the URL).

Reply:
I'm in favor of removing DEFAULT completely as it'd be more explicit.

Reply:
@philip-cline, could you move forward with removing the `DEFAULT` option and replacing with `LIST`?